modules/tinymce/src/plugins/paste/main/ts/api/Settings.ts

```diff
@@ -61,6 +62,11 @@ const getForcedRootBlockAttrs = (editor: Editor) => editor.getParam('forced_root
 
 const getTabSpaces = (editor: Editor) => editor.getParam('paste_tab_spaces', 4, 'number');
 
+const allowedImageFileTypes = (editor: Editor): string[] => {
+  const defaultImageFileTypes = 'jpeg, jpg, jpe, jfi, jfif, png, gif, bmp, webp';
+  return Tools.explode(editor.getParam('image_file_types', defaultImageFileTypes));
```

Comment:
I'd suggest ensuring this is a string by adding the 3rd "type" argument.
```suggestion
  return Tools.explode(editor.getParam('image_file_types', defaultImageFileTypes, 'string'));
```